_component_design/tabset.md

```diff
@@ -17,34 +17,95 @@ helix-ui-javascript: true
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use tabs in the following situations:
+
+- Users need to alternate between different views, while the context remains constant.
+- You want the content of a page to be scannable and easy to navigate. Tabs help users quickly find what they need.
+- You do not need to create a single, unified experience for the user. Tabs eliminate the need for the user to scroll through long pages to find the right section.
+
+Do not use tabs for the following situations:
+
+- Tabs are not suited for use for navigation
+- Tabs are not viable when users need to view content from multiple tabs simultaneously
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct tabs:
+
+- Only one active tab per page is allowed.
+- The default view must display the left-most tab.
+- Present tabs as a single row.
+- Do not construct nested tabs that include another set of tabbed content within a tab.
+- Highlight the tab corresponding to the visible content.
+- Provide all relevant contextual information and interface items so a user can perform a single task within a tab.
+- Do not use tabs for primary navigation.
+- Clearly name tabs to differentiate them. Names should be short, scannable, and preferably one word.
+- Capitalize the first letter of each word in a tab name.
+- If necessary, wrap tab names to a second line. Truncate tab names that exceed two lines.
```

Comment:
I strongly disagree with this line.  Suggest:

`Avoid using lengthy tab names.  Do not allow text to wrap or truncate.`